.vscode/settings.json

```diff
@@ -16,5 +16,11 @@
     "out": true
   },
   // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
-  "typescript.tsc.autoDetect": "off"
+  "typescript.tsc.autoDetect": "off",
+  "mdb.presetSavedConnections": [
```

Comment:
I think naming it `presetSavedConnections` is better to distinguish that this isn't overriding one's own savedConnections